Tasks/google-play-promote/package.json

```diff
@@ -11,7 +11,6 @@
   "dependencies": {
     "bluebird": "^2.10.1",
     "googleapis": "^2.1.5",
-    "minimatch": "^3.0.0",
```

Comment:
Maybe you already did it, but can you double check with Peter that vso-task-lib doesn't have a dependency on this? I believe that is why we actually added this to our tasks, even though we don't use it.